packages/btc/README.md

```diff
@@ -0,0 +1 @@
+# btc
```

Comment:
Perhaps add an intro and code small snippet

Reply:
yes sir, will update this README

Reply:
I realized we don't have README in the other packages, shall we create a small PR for adding those there? if not, I can just remove this one

Reply:
I going to dismiss this in favour of #414